src/components/SlopedBackground/index.js

```diff
@@ -1,4 +1,4 @@
-import React from "react"
```

Comment:
Please test and use following code in `src/components/SlopedBackground/index.js`:

```js
import React, { useState, useMemo } from "react"
import { css } from "@emotion/core"
import { OutboundLink } from "gatsby-plugin-google-analytics"

import BackgroundWithNavigator from "@/components/BackgroundWithNavigator"

import { background, anchor } from "./style"

const SlopedSection = ({
  children,
  slopedBackgroundColor,
  slopedBackgroundImage,
  hoverBackgroundColor,
  href,
  ...props
}) => {
  const [isHover, setIsHover] = useState(false)

  const childWithProp = useMemo(
    () =>
      React.Children.map(children, child => {
        return isHover
          ? React.cloneElement(child, { isHover: isHover })
          : children
      }),
    [children, isHover]
  )

  return (
    <BackgroundWithNavigator
      css={css`
        ${background}
        &::before {
          ${slopedBackgroundImage &&
            `background-image: ${slopedBackgroundImage};`}
          ${slopedBackgroundColor &&
            `background-color: ${slopedBackgroundColor};`}
          ${isHover &&
            hoverBackgroundColor &&
            `background-color: ${hoverBackgroundColor};`}
        }
      `}
      backgroundComponent={
        href ? (
          <OutboundLink
            href={href}
            target="_blank"
            rel="noopener noreferrer"
            onMouseEnter={() => setIsHover(true)}
            onMouseLeave={() => setIsHover(false)}
            css={anchor}
          />
        ) : null
      }
      {...props}
    >
      {childWithProp}
    </BackgroundWithNavigator>
  )
}

SlopedSection.defaultProps = {
  mode: "light",
}

export default SlopedSection
```

and below in `src/components/SlopedBackground/style.js`:
```js
import { css } from "@emotion/core"

export const background = css`
  padding-bottom: 8rem;
  width: 100%;
  position: relative;
  display: flex;
  align-items: center;
  justify-content: center;

  &::before {
    content: "";
    height: calc(100% + 2px); /** +2px to offset the edge rendering issue */
    position: absolute;
    top: 0;
    transform: skewY(-21.5deg);
    width: 100%;
  }
`

export const anchor = css`
  height: calc(100% + 2px); /** +2px to offset the edge rendering issue */
  position: absolute;
  top: 0;
  left: 0;
  bottom: 0;
  right: 0;
  transform: skewY(-21.5deg);
  width: 100%;
  z-index: 3;
`
```

Reply:
fixed.